### PR TITLE
Initial bundle setup and functionality to backup MySQL databases.

### PR DESCRIPTION
**Pull request by AndreasA**

### CODE REVIEW

File: src/Command/Backup/BackupDatabaseCommand.php [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        }
+
+        $errorOutput = $dumpProcess->getErrorOutput();
+        $io->getErrorStyle()->write($errorOutput);
+
+        $isFailure =
+            false === $dumpProcess->isSuccessful() ||
+            false === $compressProcess->isSuccessful() ||
+            // Some errors like missing PROCESS privilege result in success exit code but with corresponding error.
+            0 === strpos($errorOutput, 'mysqldump: Error: ');
```
Comment (friemt) [inline src/Command/Backup/BackupDatabaseCommand.php]: This is hander specific and should reside in the handler.

File: src/Command/Backup/BackupDatabaseCommand.php [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        }
+
+        $io->success('Finished database backup.');
+
+        return 0;
+    }
+
+    private function createNamedPipe(): string
+    {
+        $namedPipe = sprintf('%1$s/%2$s', $this->targetDirectory, bin2hex(random_bytes(32)));
```
Comment (friemt) [inline src/Command/Backup/BackupDatabaseCommand.php]: `tmpnam()` instead?

File: src/Command/Backup/BackupDatabaseCommand.php (hunk 1 above)
Comment (AndreasA) [reply src/Command/Backup/BackupDatabaseCommand.php]: already moved it 😄

File: src/Command/Backup/BackupDatabaseCommand.php [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            'DB_CONTENT_PIPE' => $namedPipe,
+            'DB_BACKUP_PATH' => $backupPath,
+        ]);
+
+        return $process;
+    }
+
+    private function getCompressCommandLineParts(): array
+    {
+        return ['bzip2', '<', '"$DB_CONTENT_PIPE"', '>', '"$DB_BACKUP_PATH"'];
```
Comment (friemt) [inline src/Command/Backup/BackupDatabaseCommand.php]: Too many lines to directly include above?

Comment (AndreasA) [reply src/Command/Backup/BackupDatabaseCommand.php]: see added comment.

File: src/Command/Backup/BackupDatabaseCommand.php (hunk 2 above)
Comment (AndreasA) [reply src/Command/Backup/BackupDatabaseCommand.php]: see added comment.

**Pull request merged**